meson.build

```diff
@@ -35,6 +35,12 @@ project(
 #   add_project_arguments(['-fvisibility=hidden'], language: 'cpp')
 # endif
 
+cpp = meson.get_compiler('cpp')
+add_project_arguments(
+    cpp.get_supported_arguments(['-Wno-misleading-indentation']),
```

Comment:
This warning was generated by the flatcc_generated header. There are a ton of these.

Perhaps there is a way to have it fixed on generation, but for now figure fine to ignore

Reply:
Do you know which version of flatcc was generated? Meson still uses flatcc as a subproject which is 0.6.1. However, the generated flatcc_generated.h file in the source tree uses some symbols that don't appear in that release. 

Should set the meson subproject up to use the same version as was vendored for CMake I think

Reply:
Ended up changing the subproject to use the same git hash. Can relook at the design in a separate PR, but the Meson subproject approach does generally make things much simpler from a build configuration standpoint